tests/integration-tests/tests/tags/test_tag_propagation.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+# import logging
+
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+
+from pcluster.utils import get_installed_version
+
+
+@pytest.mark.regions(["us-east-1"])
```

Comment:
any guidance on what region to put new tests in?

Reply:
ap-southeast-1 or 2